client/blocks/happychat/connection.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import HappychatClientApi from 'happychat-client';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { getCurrentUser } from 'state/current-user/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSkills from 'state/happychat/selectors/get-skills';
+import {
+	AUTH_TYPE_WPCOM_PROXY_IFRAME,
+	ENTRY_CHAT,
+	LAYOUT_PANEL_MAX_PARENT_SIZE,
+} from 'blocks/happychat/chat-client/constants';
+
+export default ( { state, dispatch }, {
+	nodeId = 'happychat-client',
+	layout = LAYOUT_PANEL_MAX_PARENT_SIZE,
+	minimized = true,
+	skills,
+} ) => {
+	// configure and open happychat
+	HappychatClientApi.open( {
+		authentication: {
+			type: AUTH_TYPE_WPCOM_PROXY_IFRAME,
+			options: {
+				proxy: wpcom,
+			},
+		},
+		entry: ENTRY_CHAT,
+		layout,
+		minimized,
```

Comment:
Nit: I remember we had a convention in Calypso to put properties with the same key and value names at the top of the object definition and all the other below. :)

Reply:
I wasn't aware of that, updated.